src/widgets/Footer/ui/index.tsx

```diff
@@ -1,12 +1,20 @@
+import Box from "@mui/material/Box"
+
 import { Logo } from "~/shared/ui"
+import { Theme } from "~/shared/utils"
 
 import "./style.scss"
 
 const Footer = () => {
   const buildVersion = import.meta.env.VITE_BUILD_VERSION
 
   return (
-    <footer className="d-flex justify-content-center align-items-center bg-light">
+    <Box
+      id="app-footer"
+      display="flex"
+      justifyContent="center"
+      alignItems="center"
+      sx={{ backgroundColor: Theme.colors.light.surface, borderTop: `1px solid ${Theme.colors.light.surfaceVariant}` }}>
```

Comment:
Why are some styling props passed as usual and some passed in the `sx` field?

Reply:
Because the Box component has its own properties regarding displaying and positioning, they are easy to use and read in the future. 
And all other styles I put into Sx because the Box component has not its own property for these purposes.